### PR TITLE
chore: Replace if statement with if expression

### PR DESCRIPTION
**Pull request by yezz123**
Hello MongoDB Team 👋 
I just refactored some parts of code Using some pep's rules.
Take a look and we can have a talk if there is any issues 👌🏻

### CODE REVIEW

File: test/asyncio_tests/test_examples.py [hunk 1]
```diff
@@ -902,16 +902,11 @@ async def run_transaction_with_retry(txn_coro, session):
...
-                    if exc.has_error_label("TransientTransactionError"):
-                        print("TransientTransactionError, retrying"
-                              "transaction ...")
-                        continue
-                    else:
+                    print('Transaction aborted. Caught exception during transaction.')
+                    if not exc.has_error_label('TransientTransactionError'):
                         raise
+                    print('TransientTransactionError, retryingtransaction ...')
+                    continue
```
Comment (ShaneHarvey) [inline test/asyncio_tests/test_examples.py:909]: For this file I prefer the old code since it more easily fits in 80 chars. What was the pep issue here? Could you revert it?

Could you also fix the missing space in the "retryingtransaction" error message?

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]
What tool did you use to find these issues? Or was this a manual audit?

File: test/asyncio_tests/test_examples.py (hunk 1 above)
Comment (yezz123) [reply test/asyncio_tests/test_examples.py:909]: I don't use any tool, just some Manual refactoring (old school methods docs and some references), i will fix this and then you can re-review it

Comment (yezz123) [reply test/asyncio_tests/test_examples.py]: @ShaneHarvey fixed

File: test/asyncio_tests/test_examples.py [hunk 2]
```diff
@@ -974,13 +974,13 @@ async def run_transaction_with_retry(txn_coro, session):
                     break
                 except (ConnectionFailure, OperationFailure) as exc:
                     # If transient error, retry the whole transaction
-                    if exc.has_error_label("TransientTransactionError"):
-                        print("TransientTransactionError, retrying "
-                              "transaction ...")
-                        continue
-                    else:
+                    if not exc.has_error_label("TransientTransactionError"):
```
Comment (ShaneHarvey) [inline test/asyncio_tests/test_examples.py:977]: Can you revert these changes as well? Same reason as before: it doesn't fit in 80 chars.

Once you revert, feel free to replace the "else: raise" with just "raise". That would clean it up a bit.

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: test/asyncio_tests/test_examples.py [hunk 3]
```diff
@@ -1177,4 +1177,4 @@ def strptime(s):
 
         # Start Versioned API Example 8
         # 8
-        # End Versioned API Example 8
+        # End Versioned API Example 8
```
Comment (ShaneHarvey) [inline test/asyncio_tests/test_examples.py:1180]: Please undo this change as well.

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]

File: test/asyncio_tests/test_examples.py (hunk 3 above)
Comment (yezz123) [reply test/asyncio_tests/test_examples.py]: done

**Comment by ShaneHarvey** [comment]
Thanks! I'm running the test suite now. Planning to merge once the tests pass.
